_posts/2019-01-03-elixir-kaffe-codealong.md

```diff
@@ -0,0 +1,204 @@
+# Connecting Elixir to Kafka with Kaffe: A codealong
+_The most basic example that exists_
+
+If we want to use the popular messaging system Kafka with our Elixir projects, we have a few wrappers we can choose from. One of them, [Kaffe](https://github.com/spreedly/kaffe), proved difficult to troubleshoot. Just try googling "Elixir Kaffe" yourself.
```

Comment:
Should we elaborate on why given a few options to choose from, we're going to cover this one which has proved difficult to troubleshoot? Is our intent to provide a missing resource or despite it's difficulties, it is a better option because of ______

Reply:
Definitely to provide a missing resource, I haven't seen that it's a better option than `kafka_ex`. I can elaborate on that.